templates/semantic-ui/invenio_app_rdm/footer.html

```diff
@@ -8,81 +8,76 @@
-              <p><a href="{{invenio_rdm_gh}}">GitHub</a></p>
-              {% endtrans %}
-              {% trans invenio_rdm_milestones="https://github.com/inveniosoftware/product-rdm" %}
-              <p><a href="{{invenio_rdm_milestones}}">Project Milestones</a></p>
-              {% endtrans %}
-              {% trans invenio_rdm_docs_site="http://inveniordm.docs.cern.ch" %}
-              <p><a href="{{invenio_rdm_docs_site}}">Documentation</a></p>
-              {% endtrans %}
-            </div>
-            <div class="rdm-footer five wide tablet two wide computer column">
-              <h2 class="ui small header">{{_ ("Community")}}</h2>
-              {% trans invenio_rdm_chatroom="https://discord.gg/8qatqBC" %}
-              <p><a href="{{invenio_rdm_chatroom}}">Chatroom</a></p>
-              {% endtrans %}
-              {% trans invenio_rdm_talk="https://invenio-talk.web.cern.ch/c/projects/invenio-rdm" %}
-              <p><a href="{{invenio_rdm_talk}}">Forum</a></p>
-              {% endtrans %}
-              {% trans invenio_events="https://inveniosoftware.org/events/" %}
-              <p><a href="{{invenio_events}}">Events & training</a></p>
-              {% endtrans %}
-            </div>
+          <div class="rdm-footer five wide tablet three wide computer eight wide mobile column">
+            <h2 class="ui small header">{{ _("About") }}</h2>
+            {% trans about="/about" %}
+              <p><a href="{{ about }}">About</a></p>
+            {% endtrans %}
+            {% trans blog="https://cds-blog.web.cern.ch/" %}
+              <p><a href="{{ blog }}">Blog</a></p>
+            {% endtrans %}
+            {% trans privacy="/privacy-policy" %}
+              <p><a href="{{ privacy }}">Privacy notice</a></p>
+            {% endtrans %}
+          </div>
+          <div class="rdm-footer five wide tablet three wide computer eight wide mobile column">
+            <h2 class="ui small header">{{ _("Help") }}</h2>
+            {% trans mattermost="https://mattermost.web.cern.ch/it-dep/channels/cds" %}
+              <p><a href="{{ mattermost }}">Mattermost (CERN login)</a></p>
+            {% endtrans %}
+            {% trans snow="https://cern.service-now.com/service-portal/function.do?name=CDS" %}
+              <p><a href="{{ snow }}">CERN Service Now</a></p>
+            {% endtrans %}
+          </div>
+          <div class="rdm-footer five wide tablet three wide computer eight wide mobile column">
+            <h2 class="ui small header">{{ _("Other CDS websites") }}</h2>
+            {% trans videos="https://videos.cern.ch" %}
+              <p><a href="{{ videos }}">videos.cern.ch</a></p>
+            {% endtrans %}
+            {% trans library="https://catalogue.library.cern.ch" %}
+              <p><a href="{{ library }}">CERN Library Catalogue</a></p>
+            {% endtrans %}
+            {% trans old_version="https://cds.cern.ch" %}
+              <p><a href="{{ old_version }}">Previous CDS</a></p>
```

Comment:
```suggestion
              <p><a href="{{ old_version }}">Old CDS</a></p>
```
as opposite to `new-cds.cern.ch`?

Reply:
I didn't want to say "old" because it is not very formal word

Reply:
![Screenshot 2023-06-07 at 15 19 45](https://github.com/CERNDocumentServer/cds-rdm/assets/2089455/d8962ad5-64e6-47d6-bc55-378e52dfcd05)